.github/workflows/run-core-traits-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: [3.6, 3.9, 3.10-dev]
+        python-version: [3.6, 3.8, 3.10.0-rc.2]
```

Comment:
I guess we're running the full testsuite with all supported pythons - which is why we can just ignore 3.7 and 3.9 here?

Reply:
Good point; we should update this too, I think. (To include all Python versions.)